analyzers/src/SonarAnalyzer.CSharp/Rules/ImplementSerializationMethodsCorrectly.cs

```diff
@@ -48,5 +51,29 @@ public sealed class ImplementSerializationMethodsCorrectly : ImplementSerializat
                 .FirstOrDefault()
                 ?.Identifier
                 .GetLocation();
+
+        protected override void Initialize(SonarAnalysisContext context)
+        {
+            context.RegisterSyntaxNodeActionInNonGenerated(c =>
+                                                           {
```

Comment:
I find this indenting exaggerating - are we expected to change this to _all_ classes? what's the advantage here? 

Please revert. This is introducing a style change and making our code inconsistent and harder to read.